shared/document_sync.tf

```
@@ -0,0 +1,42 @@
+locals {
+  sirius_root       = "arn:aws:iam::${local.account.sirius_account_id}:root"
+  sirius_env_lambda = "arn:aws:iam::${local.account.sirius_account_id}:role/sirius-reports-${local.account.name}-v1"
+}
+
+resource "aws_iam_role" "integrations_s3_read" {
+  assume_role_policy = data.aws_iam_policy_document.integrations_assume_role.json
+  name               = "integrations-s3-read-${local.account.name}"
+  tags               = local.default_tags
+}
+
+data "aws_iam_policy_document" "integrations_assume_role" {
+  statement {
+    sid    = "AllowIntegrationsLambdaS3"
+    effect = "Allow"
+    principals {
+      type        = "AWS"
+      identifiers = local.account.development ? [local.sirius_root] : [local.sirius_env_lambda]
```

```suggestion
      identifiers = local.account.name = "development" ? [local.sirius_root] : [local.sirius_env_lambda]
```

Saves having to add a `development` prop to all accounts then?

---

I did try that as it would have been nicer. Unfortunately terraform doesn't like that sort of statement. It gives a syntax error.

---

Ok, I missed out on this. It does like it with a == and I just must have missed it!